### DISCUSSION TITLE
No commit pushedDate could be found

### DISCUSSION THREAD

**Discussion by borque** [microsoft/winget-pkgs#111665] [General] [answered]
Hello!
I have noticed that some of my recent submissions have encountered the above error.
Is there something wrong with the publishing pipelines at the moment?

**Comment by mdanish-kh** [comment, marked as answer]
Being tracked in #111634

The pipelines have been little unstable past couple of days

**Reply by BrandonWanHuanSheng** [reply]
It is true and I also noticed that thing which is the issue coming from [azure-pipelines[bot]](https://github.com/apps/azure-pipelines)

**Reply by borque** [reply]
Thanks!
